deployments/apiv2/services/users/src/main.py

```diff
@@ -154,6 +160,9 @@ async def login(request: Request, details: Union[UserLogin, CustomerLogin]):
                     _, customer_tier = split_scope_account_data(scope[0])
                     scope[0] = f"customer:{customer_tier}"
 
+                # after login flow succes update the last-login filed of the user
```

Comment:
typo: success. Also not sure if "filed" is supposed to be "field", but this whole comment should probably read: if login was successful, that update last_login column to now